app/helpers/fhir_helper.rb

```diff
@@ -85,7 +90,9 @@ def patient_as_fhir(patient)
         to_us_core_race(races_as_hash(patient)),
         to_us_core_ethnicity(patient.ethnicity),
         to_us_core_birthsex(patient.sex),
-        to_latest_transfer_extension(patient),
+        *to_transfer_extensions(patient),
```

Comment:
We do need to load the transfers, because the requirements are that all of the data that is exported in Excel be available. This is only ever really an issue when searching for patients, since that is the case that could lead to N+1 transfers being loaded, so I think I was able to solve this in https://github.com/SaraAlert/SaraAlert/pull/1016/commits/8cb31978737170940f728bd54fef61b4fe51c20b by eager loading transfers and their needed associations.

Reply:
I tested these changes it the performance does seem to be better. Out of curiosity though, is there a reason why we aren't doing transfer searches via a separate endpoint similar to all the other associated records (assessments/labs/vaccines/etc) and including it directly on the patient search itself?

Reply:
The main reason is that there really isn't any good mapping between a [FHIR Resource](https://www.hl7.org/fhir/resourcelist.html) and a transfer. In the case of something like vaccine, that very easily maps to a [FHIR Immunization](https://www.hl7.org/fhir/immunization.html). But there really isn't anything that maps well to a transfer, so we have to do it via an extension.

Reply:
that makes sense, thanks Nick!